i18n/sync_tx_translations.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * @fileoverview
+ * Script to pull translations for blocks from transifex and generate the scratch_msgs file.
+ * Expects that the project and resource have already been defined in Transifex, and that
+ * the person running the script has the the TX_TOKEN environment variable set to an api
+ * token that has developer access.
+ */
+
+// Fail immediately if the TX_TOKEN is not defined
+if (!process.env.TX_TOKEN) {
+  console.error(new Error('You must be a Scratch Team member and set your TX_TOKEN to sync with Transifex'));
```

Comment:
Same goes here about how to supply `TX_TOKEN`, and maybe we should explain this error via a "Usage" message as well?